tests/test_workspaces.rs

```diff
@@ -122,17 +122,21 @@ fn test_workspaces_conflicting_edits() {
     Hint: Run `jj workspace update-stale` to update it.
     "###);
     let stdout = test_env.jj_cmd_success(&secondary_path, &["workspace", "update-stale"]);
-    // It was detected that the working copy is now stale
-    // TODO: Since there was an uncommitted change in the working copy, it should
+    // It was detected that the working copy is now stale.
+    // Since there was an uncommitted change in the working copy, it should
     // have been committed first (causing divergence)
     insta::assert_snapshot!(stdout, @r###"
+    Concurrent modification detected, resolving automatically.
```

Comment:
Do you think it would be useful to also show the operation log?

Reply:
Might be useful, but I think showing a hint like "(Run `jj op log`)" is good enough.